supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -1,10 +1,59 @@
+[[style-advice-for-release-note-texts]]
+== Style advice for release note texts
+
+The normal stylistic guidelines for documentation from IBM Style and the RH SSG apply also to release note texts, particularly the following:
+
+* When you first mention a utility, package, command, or similar item outside of a heading, define it. Do not assume that the customer is familiar with it. 
+* Omit the definition in later occurrences. If the context is ambiguous, for example, when the release note text mentions both an `example` package and an `example` service, you can repeat the definition to add clarity.
+* Avoid definitions in headings, but you can use them to disambiguate different meanings of the same name.
+* Do not start a sentence with a word in lower case. You can repeat a definition to avoid starting a sentence with a lowercase name. See: link:https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#general-text[Capitalization (IBM Style)].
+* Expand abbreviations in descriptions. Do not expand abbreviations in headings. See: link:https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations[Abbreviations (IBM Style)].
+* When comparing versions, use "later" and "earlier" instead of "newer" and "older". See: link:https://www.ibm.com/docs/en/ibm-style?topic=word-usage#later[Later (IBM Style)].
+* Avoid complicated syntax, such as passive voice and modal verbs, and ambiguous language. For example, replace "Should XY happen" with "If XY happens".
+* Keep admonitions to a minimum, and avoid placing multiple admonitions in a single note. Do not begin a release note with an admonition. See xref:admonitions[Admonitions (RH SSG)].
+* Avoid using telegraphic language common in merge requests and changelogs, for example, "Remove deprecated support macros".
+
+[[tenses-in-release-notes]]
+== Tenses in release notes
+
+Write the release notes from the perspective of just after the release, which is when most of the customers will read them. The state before the update is in the past, the state after the update is in the present, any future updates are in the future.
+
+* Use the *simple present tense* as much as possible.
+* Do not use *future tenses* (or "should" or "might") to describe the state after the update. You can use future tenses to refer to future releases.
+* Use the *simple past tense* to describe the previous situation before the current update.
+* You can use the *present perfect tense* to describe the current update. “With this update, the X component has been removed from the Y package.”
+* Follow the CCFR (Cause-Consequence-Fix-Result) tense logic in bug fixes.
```

Comment:
And CCWR for known issues: 

    Cause – the user action or circumstances that trigger the bug
    Consequence – what the user experience is when the bug occurs
    Workaround - if available
    Result – what happens now that the patch is applied

Reply:
Yes, it makes sense to include known issues and explain what Cause-Consequence-Fix-Result means in addition to spelling out the acronym.

Reply:
Darragh, I'm not sure that the CCWR has a _tense logic_ per se; all of those are typically in the present tense, so it does not make much sense to me to include it here in the Tenses section.

I'm all for including the CCWR breakdown more specifically in the Known issues section later.